### PR TITLE
Feat: Dutch language (nl)

### PR DESCRIPTION
**Pull request by Hugos68**
Closes: #437

Added the Dutch language

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you for this PR! Do you know another native speaking developer that could review the translations for me?

File: packages/i18n/src/nl.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    endsWith:       (issue) => `Ongeldig einde: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    equal:          (issue) => `Ongeldige waarde: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    excludes:       (issue) => `Ongeldige inhoud: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    finite:         (issue) => `Ongeldige eindige: ${issue.received} ontvangen`,
+    hash:           (issue) => `Ongeldige hash: ${issue.received} ontvangen`,
+    hexColor:       (issue) => `Ongeldige hex kleur: ${issue.received} ontvangen`,
+    hexadecimal:    (issue) => `Ongeldige hexadecimaal: ${issue.received} ontvangen`,
+    imei:           (issue) => `Ongeldige IMEI: ${issue.received} ontvangen`,
+    includes:       (issue) => `Ongeldige inhoud: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    integer:        (issue) => `Ongeldige geheel getal: ${issue.received} ontvangen`,
```
Comment (nikolai-cc) [inline packages/i18n/src/nl.ts:26]: “ongeldig geheel getal”

File: packages/i18n/src/nl.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    minLength:      (issue) => `Ongeldige lengte: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    minSize:        (issue) => `Ongeldige grootte: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    minValue:       (issue) => `Ongeldige waarde: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    multipleOf:     (issue) => `Ongeldige meervoudige: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    notBytes:       (issue) => `Ongeldige bytes: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    notLength:      (issue) => `Ongeldige lengte: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    notSize:        (issue) => `Ongeldige grootte: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    notValue:       (issue) => `Ongeldige waarde: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
+    octal:          (issue) => `Ongeldige octaal: ${issue.received} ontvangen`,
+    regex:          (issue) => `Ongeldige formaat: ${issue.expected} verwacht, maar ${issue.received} ontvangen`,
```
Comment (nikolai-cc) [inline packages/i18n/src/nl.ts:55]: “Ongeldig formaat”

**Review by nikolai-cc — COMMENTED** [review]
twee e’tjes eraf (regel 26 en 55) en verder lijkt het me top

File: packages/i18n/src/nl.ts (hunk 2 above)
Comment (Hugos68) [reply packages/i18n/src/nl.ts:55]: Bedankt!

File: packages/i18n/src/nl.ts (hunk 1 above)
Comment (Hugos68) [reply packages/i18n/src/nl.ts:26]: Bedankt!

**Comment by Hugos68** [comment]
Thanks @nikolai-cc for the peer review. I've gone ahead and implemented your suggested changes.

**Comment by fabian-hiller** [comment]
Is this PR ready to be merged or should we wait for another native speaker's review?

**Comment by nikolai-cc** [comment]
IMO with those minor changes it's good to go

**Comment by Hugos68** [comment]
lgtm

**Comment by fabian-hiller** [comment]
Thank you both very much! I plan to merge it tomorrow. 🧩

**Comment by fabian-hiller** [comment]
v0.6.0 is available

**Pull request merged**